src/templates/Events.js

```diff
@@ -1,4 +1,19 @@
 import {graphql} from 'gatsby';
+import {useEffect} from 'react';
+import {
+  values,
+  mapObjIndexed,
+  map,
+  filter as filterRamda,
```

Comment:
This is manually fixed to pass webpack's tree shaking. We cannot import `filter` from `~/contexts` unless we import ramda's `filter` by alias.